Tests/Filter/DateFilterTest.php

```diff
@@ -20,7 +22,7 @@
  */
 class DateFilterTest extends TestCase
 {
-    public function testGetFilterTypeClass()
+    public function testGetFilterTypeClass(): void
```

Comment:
You need to avoid this by removing all the php IFs but the 5.6 one in `.php_cs.cache`

Reply:
I don't really get what you mean 😄

Reply:
Sorry wrong file, I meant `.php_cs.dist`. This one possibly only appears on the dev-kit branch

Reply:
cc @Soullivaneuh